vite.config.ts

```diff
@@ -3,20 +3,14 @@ import react from '@vitejs/plugin-react';
 import viteTsconfigPaths from 'vite-tsconfig-paths';
 import svgrPlugin from 'vite-plugin-svgr';
 import { sentryVitePlugin } from '@sentry/vite-plugin';
-import pluginRewriteAll from 'vite-plugin-rewrite-all';
 
 // https://vitejs.dev/config/
 export default defineConfig(({ mode }) => {
   // Load env file based on `mode` in the current working directory.
   // Set the third parameter to '' to load all env regardless of the `VITE_` prefix.
   const env = loadEnv(mode, process.cwd(), '');
 
-  const plugins: PluginOption[] = [
-    react(),
-    viteTsconfigPaths(),
-    svgrPlugin(),
-    pluginRewriteAll(),
-  ];
+  const plugins: PluginOption[] = [react(), viteTsconfigPaths(), svgrPlugin()];
```

Comment:
Do you know what `pluginRewriteAll` was doing ? Maybe @pingustar you know ?

Reply:
I added it - it's used on dev server to not rewrite the path if it includes a dot '.'. This was useful for explorer search by ens as all the links would have a dot in the url and we couldn't test it locally.

Package here: https://github.com/ivesia/vite-plugin-rewrite-all

This is no longer required, check here:

https://github.com/vitejs/vite/pull/14142